modules/restore-unauthenticated-group-bindings.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// * authentication/???.adoc
+// * post_installation_configuration/???.adoc
```

Comment:
Not sure, where to add it or if I should create entries completely. I would assume it would make sense to have it in authenticadtion / post_installation_configuration.